modules/odr_api/Taskfile.yml

```diff
@@ -15,3 +15,8 @@ tasks:
     cmds:
       - cmd: python -m debugpy --listen 0.0.0.0:5678 --wait-for-client ./modules/odr_api/odr_api/main.py
         platforms: [windows, linux, darwin]
+
+  test_api:
```

Comment:
This should be included in the api-test taskfile https://github.com/Open-Model-Initiative/OMI-Data-Pipeline/blob/main/modules/odr_api/tests/Taskfile.yml . 

To keep naming in alignment across tasks use - and not _ in task names, (change to test-api). I would also just rename this one to health-check and add it as the first command ran in the test-all command in the odr_api/test/Taskfile.yml

Reply:
Seems like something is a bit off when running this command when in the root directory: 

```
PS C:\OMI-Data-Pipeline> task api:test_api
task: [api:test_api] pytest -s modules/odr_api/tests/integration/models/test_health.py
========================================= test session starts ==========================================
platform win32 -- Python 3.12.0, pytest-8.3.2, pluggy-1.5.0
rootdir: C:\OMI-Data-Pipeline\modules\odr_api
configfile: pyproject.toml
plugins: anyio-4.4.0
collected 0 items / 1 error

================================================ ERRORS ================================================ 
_______________________ ERROR collecting tests/integration/models/test_health.py _______________________ 
ImportError while importing test module 'C:\OMI-Data-Pipeline\modules\odr_api\tests\integration\models\test_health.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
C:\AppData\Local\Programs\Python\Python312\Lib\importlib\__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
modules\odr_api\tests\integration\models\__init__.py:1: in <module>
    from models.test_users import TestUserLifecycle
E   ModuleNotFoundError: No module named 'models'
======================================= short test summary info ======================================== 
ERROR modules\odr_api\tests\integration\models\test_health.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 
=========================================== 1 error in 0.08s =========================================== 
task: Failed to run task "api:test_api": exit status 2
PS C:\OMI-Data-Pipeline>
```